### PR TITLE
Pretty-print certificates from the notary CLI command `notary cert list`

### PR DESCRIPTION
**Pull request by cyli**
Signed-off-by: Ying Li <ying.li@docker.com>

Output looks like:

```
  GUN                FINGERPRINT OF TRUSTED ROOT CERTIFICATE                EXPIRES IN  
----------------------------------------------------------------------------------------
  repo   0bb218afd3e64cf29622948cc1b5e5a7b0b306e2cbe0617f5092fc3dc89927ec   3649 days   
```

If there are no certs:

```
No trusted root certificates present.
```

### CODE REVIEW

**Comment by endophage** [comment]
I feel like we should move all our pretty printing functions into their own file within the package... thoughts?

**Comment by cyli** [comment]
I have no strong opinion, but would be happy to do so :)

**Comment by endophage** [comment]
I think it would be good. We're in the region of 150 loc of pretty printing code. Seems like a sizable enough chunk of cohesive functionality that it deserves its own file.

File: cmd/notary/prettyprint_test.go [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+		{role: data.CanonicalRootRole, gun: "", keyID: "a", location: "j"},
+		{role: data.CanonicalRootRole, gun: "", keyID: "z", location: "z"},
+		{role: "a", gun: "a", keyID: "a", location: "y"},
+		{role: "b", gun: "a", keyID: "a", location: "y"},
+		{role: "b", gun: "a", keyID: "b", location: "y"},
+		{role: "b", gun: "a", keyID: "b", location: "z"},
+		{role: "a", gun: "b", keyID: "a", location: "z"},
+	}
+	jumbled := make([]keyInfo, len(expected))
+	// randomish indices
```
Comment (endophage) [inline cmd/notary/prettyprint_test.go]: Did you use a d8? :-P

**Comment by endophage** [comment]
LGTM

File: cmd/notary/prettyprint_test.go (hunk 1 above)
Comment (cyli) [inline cmd/notary/prettyprint_test.go]: Heh, no, hence random-ish.  :)

**Pull request merged**